Source/WebCore/style/AnchorPositionEvaluator.h

```diff
@@ -50,17 +50,20 @@ enum class AnchorPositionResolutionStage : uint8_t {
     Positioned,
 };
 
-using AnchorElements = HashMap<AtomString, WeakRef<Element, WeakPtrImplWithEventTargetData>>;
+using AnchorName = ScopedName;
+using AnchorNameOrAuto = std::optional<ScopedName>;
```

Comment:
I find these aliases to add more confusion/inconsistency than clarification personally. I'd rather use ScopedName directly here.

Not to mention the extra unnecessary includes.

Reply:
I got rid of the aliases and just use `ScopedName` or `std::optional<ScopedName>` as appropriate.